### PR TITLE
Complete db seeding

### PR DESCRIPTION
**Pull request by amrutapatil1720**

### CODE REVIEW

File: client/src/components/index.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './app.jsx';
+
+ReactDOM.render(
+   <App />,
```
Comment (elchangtw) [inline client/src/components/index.jsx]: May want to change main component name to your module name at a later date?

File: client/src/components/reservationForm.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                    <div className="input_date">
+                        <span className="date_span">
+                        <i class="fa fa-calendar" aria-hidden="true"></i>
+
+                        </span>
+                            <input placeholder="YYYY-M-D" value="Sun, Dec 1" className="input"></input>
+                       
+                    </div>
+
+
```
Comment (elchangtw) [inline client/src/components/reservationForm.jsx]: Mildly confusing spacing ._.?

File: database/model.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: '',
+  database: 'yelp',
+});
+
+connection.connect();
+
+module.exports ={
```
Comment (elchangtw) [inline database/model.js]: i think missing a space? not big issue.

File: database/schema.sql [hunk 4]
```diff
@@ -0,0 +1,28 @@
+ DROP DATABASE IF EXISTS yelp;
+ 
+ CREATE DATABASE yelp;
+ use yelp;
+  
+  CREATE TABLE restaurant (
```
Comment (elchangtw) [inline database/schema.sql]: Not sure if too nitpicky, but indent xd

File: database/seed.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    database: 'yelp'
+  });
+  
+  connection.connect();
+
+module.exports={
+    seedRestaurant: () => {
+        const list = [];
+        console.log(restaurants.length-1);
+        for (var i = 0; i <= 100; i++) {
```
Comment (elchangtw) [inline database/seed.js]: Will result in 101 iterations: intended result?

Comment (amrutapatil1720) [reply database/seed.js]: fixed

File: client/src/components/reservationForm.jsx (hunk 2 above)
Comment (amrutapatil1720) [reply client/src/components/reservationForm.jsx]: will do it when working on frontend side

**Pull request merged**